tests/SushiTest.php

```diff
@@ -2,104 +2,157 @@
-
-        Bar::count();
-    }
-
-    /** @test */
-    function uses_in_memory_if_the_cache_directory_is_not_writeable_or_not_found()
+    public function it_generates_correct_schema()
     {
-        config(['sushi.cache-path' => $path = __DIR__.'/non-existant-path']);
-
-        Foo::count();
-
-        $this->assertFalse(file_exists($path));
-        $this->assertEquals(':memory:', (new Foo)->getConnection()->getDatabaseName());
+        $table = new Blueprint('table');
+
+        Maki::getSchema($table, new Maki(), Maki::getRows()->first());
+
+        $columns = collect($table->getColumns());
+
+        $id = $columns->firstWhere('name', 'id');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'bigInteger',
+            'name' => 'id',
+            'autoIncrement' => true,
+            'unsigned' => true,
+        ], $id);
+
+        $name = $columns->firstWhere('name', 'name');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'string',
+            'name' => 'name',
+            'nullable' => true,
+        ], $name);
+
+        $age = $columns->firstWhere('name', 'age');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'integer',
+            'name' => 'age',
+            'nullable' => true,
+        ], $age);
+
+        $price = $columns->firstWhere('name', 'price');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'float',
+            'name' => 'price',
+            'nullable' => true,
+        ], $price);
+
+        $properties = $columns->firstWhere('name', 'properties');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'json',
+            'name' => 'properties',
+            'nullable' => true,
+        ], $properties);
+
+        $publishedAt = $columns->firstWhere('name', 'published_at');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'timestamp',
+            'name' => 'published_at',
+            'nullable' => true,
+        ], $publishedAt);
+
+        $createdAt = $columns->firstWhere('name', 'created_at');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'timestamp',
+            'name' => 'created_at',
+            'nullable' => true,
+        ], $createdAt);
+
+        $updatedAt = $columns->firstWhere('name', 'updated_at');
+        $this->assertEqualsColumnDefinition([
+            'type' => 'timestamp',
+            'name' => 'updated_at',
+            'nullable' => true,
+        ], $updatedAt);
     }
 
     /** @test */
-    function caches_sqlite_file_if_storage_cache_folder_is_available()
+    public function it_inserts_all_rows()
     {
-        Foo::count();
-
-        $this->assertTrue(file_exists($this->cachePath));
-        $this->assertStringContainsString(
-            'sushi/tests/cache/sushi-tests-foo.sqlite',
-            (new Foo)->getConnection()->getDatabaseName()
-        );
+        $this->assertEquals(101, Maki::count());
```

Comment:
Can it be `'101'` as a string? If not the type check should be enforced with `assertSame`

Reply:
https://github.com/calebporzio/sushi/pull/7/commits/29d0dc1137f1bb10ddbc30567759ac3b29031c7d